sdk/search/azure-search-documents/src/main/java/module-info.java

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 module com.azure.search.documents {
-    requires transitive com.azure.core.serializer.json.jackson;
+    requires transitive com.azure.core;
```

Comment:
Since `com.azure.core.serializer.json.jackson` transitively requires `com.azure.core`, the previous definition of module-info should technically work too. But search does not use any of the core Jackson types directly, so, this change looks good to me.